docker/docker-create-container.sh

```diff
@@ -2,8 +2,13 @@
 
 QNX_SDP_VERSION=qnx800
 
-docker run -it \
+docker create -it --name=qnx-build \
```

Comment:
Suggest changing the container name here, to avoid conflicting with the one in the repo  in gitlab.
```suggestion
docker create -it --name=qnx-ports-build \
```